auto-value-gson/gradle.properties

```diff
@@ -0,0 +1,20 @@
+GROUP=com.ryanharter.auto.value
```

Comment:
this module specific gradle.properties can keep the `POM_ARTIFACT/POM_NAME/POM_PACKAGING` directives.